src/Components/ApplicationCard.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import '../Styles/ApplicationCard.scss';
 
-const ApplicationCard = () => {
+const ApplicationCard = ({name, email, description}) => {
```

Comment:
Pulling in the real data! This is exciting. And it works. Which is even more exciting.